sdk/storage/examples/blob_04.rs

```diff
@@ -30,12 +31,12 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let data = b"1337 azure blob test";
     let mut block_ids = Vec::new();
     for (i, block) in data.chunks(64 * 1024 * 1024 /* 64 MiB */).enumerate() {
-        block_ids.push(i.to_be_bytes());
+        block_ids.push(Bytes::from(format!("{}", i)));
         let hash = md5::compute(block).into();
-        let block_id = (&i.to_be_bytes() as &[u8]).into();
+        let block_id = Bytes::from(format!("{}", i));
```

Comment:
nit: cloning the `block_id` from above is probably simpler and slightly more efficient than doing the formatting twice.